apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingWithdraws/StakingWithdraws.types.ts

```diff
@@ -0,0 +1,12 @@
+export type StakingWithdrawnItem = {
```

Comment:
When I said to rename the main component and its folder, I meant all types, files, queries, hooks, etc. So this should be `StakingWithdrawItem` for instance. Pls, rename the whole folder accordingly.